src/custom.js

```diff
@@ -18,4 +19,10 @@ const custom = step => {
     .withViews(...templates, stepInstance.dirname);
 };
 
-module.exports = custom;
+const verifyResult = (step, methodName, expectedContent, session = {}) => {
```

Comment:
i wonder if it would be more understandable if it was called `stepAsInstance` and just returned the step instance which devs could run their tests on?

Reply:
what do you think?

Reply:
> what do you think?

Agreed. Removed custom method and added stepAsInstance.